clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -617,7 +617,7 @@ public synchronized void updatePreferredReadReplica(TopicPartition tp, int prefe
      * Unset the preferred read replica. This causes the fetcher to go back to the leader for fetches.
      *
      * @param tp The topic partition
-     * @return true if the preferred read replica was set, false otherwise.
+     * @return return the removed preferred read replica was set, None otherwise.
```

Comment:
we have 2 `return` now. How about:
`@return the removed preferred read replica if set, None otherwise.`

Reply:
@showuon I have updated the code. Thanks for your review.